### PR TITLE
Update knative-build-pipelines

### PR DESCRIPTION
**Pull request by arilivigni**
- Updated to 4.0 image stream

### CODE REVIEW

File: ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  name: origin-v3.11
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
 base_images:
   base:
     cluster: https://api.ci.openshift.org
-    name: origin-v3.11
+    name: '4.0'
     namespace: openshift
```
Comment (bbrowning) [inline ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml]: I believe the namespace needs to be `ocp` here as well

File: ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml [hunk 2]
```diff
@@ -1,11 +1,10 @@
 tag_specification:
-  cluster: https://api.ci.openshift.org
-  name: origin-v3.11
-  namespace: openshift
+  name: '4.0'
+  namespace: ocp
 base_images:
   base:
     cluster: https://api.ci.openshift.org
```
Comment (bbrowning) [inline ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml]: I don't believe cluster is strictly needed here. Others don't seem to have it. See https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml as an example

Comment (arilivigni) [reply ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml]: Please check now

File: ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml (hunk 1 above)
Comment (arilivigni) [reply ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml]: Please check now

**Pull request merged**